### PR TITLE
Add Development Process Guidelines base

### PR DESCRIPTION
**Pull request by rexim**
Split #7

### Remaining Work

- Release Process
- Issue Tracker Hygiene
- Problem Reporting Format
- Label System
- Contribution Protocol
- Issue & PR templates

### CODE REVIEW

File: DevelopmentProcess.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Development Process Guidelines
+
+These guidelines are indended to be used by [@morganey-lang] community
+members while working on [@morganey-lang]'s projects.
+
+They are completely optional, because [per rules](README.md#becoming-a-project-leader):
+
+> The Project Leaders are responsible for organizing the development process of the projects they lead.
+
+So, before contributing anything to any of the [@morganey-lang]'s projects, ask a corresponding Project Leader about development process details. See [Projects.md](Projects.md) for the full list of [@morganey-lang] projects and their corresponding Project Leaders.
```
Comment (ForNeVeR) [inline DevelopmentProcess.md:10]: Probably add a line addressed to a project leader, e.g. "if you're a project leader, consider following these practices with your development".

File: DevelopmentProcess.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+f
```
Comment (ForNeVeR) [inline DevelopmentProcess.md]: I think that's a typo.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
I'm not sure if we want this merged with all these "TBD" marks. It has no actual value without at least some of them filled in.

File: DevelopmentProcess.md (hunk 2 above)
Comment (rexim) [reply DevelopmentProcess.md]: Fxd

File: DevelopmentProcess.md (hunk 1 above)
Comment (rexim) [reply DevelopmentProcess.md:10]: Add a small notice about that

**Comment by rexim** [comment]
> I'm not sure if we want this merged with all these "TBD" marks. It has no actual value without at least some of them filled in.

Once this PR is approved I'm gonna fill them with corresponding issue numbers

**Review by ForNeVeR — APPROVED** [review]
Okay then. LGTM!

**Comment by rexim** [comment]
@ForNeVeR thanks for the review!

**Pull request merged**